### PR TITLE
Audit Changes

### PR DESCRIPTION
**Pull request by hexonaut**
This will switch from LTV0 and freezing to just disabling borrows on all assets.

### CODE REVIEW

**Review by barrutko — DISMISSED** [review]
LGTM

File: test/KillSwitchOracleIntegration.t.sol [hunk 1]
```diff
@@ -76,212 +79,60 @@ contract KillSwitchOracleIntegrationTest is Test {
...
-        assertEq(configuration.getFrozen(),               frozen);
-        assertEq(configuration.getPaused(),               paused);
-        assertEq(configuration.getLtv(),                  ltv);
-        assertEq(configuration.getLiquidationThreshold(), liquidationThreshold);
-        assertEq(configuration.getLiquidationBonus(),     liquidationBonus);
+        vm.startPrank(DAI_BORROWER_WALLET);
+        IERC20(DAI).approve(address(pool), 1e18);
+        pool.repay(DAI, 1e18, 2, DAI_BORROWER_WALLET);
+        vm.stopPrank();
+
```
[the remark this answers is not in the document]
Comment (hexonaut) [reply test/KillSwitchOracleIntegration.t.sol:149]: done

File: test/KillSwitchOracleIntegration.t.sol [hunk 2]
```diff
@@ -76,212 +79,79 @@ contract KillSwitchOracleIntegrationTest is Test {
...
-        assertEq(configuration.getLiquidationThreshold(), liquidationThreshold);
-        assertEq(configuration.getLiquidationBonus(),     liquidationBonus);
+        vm.startPrank(DAI_BORROWER_WALLET);
+
+        // Make sure we can repay
+        IERC20(DAI).approve(address(pool), 1e18);
+        pool.repay(DAI, 1e18, 2, DAI_BORROWER_WALLET);
+
+        // Borrow should revert on all assets
+        vm.expectRevert(bytes('30'));  // BORROWING_NOT_ENABLED
```
Comment (lucas-manuel) [inline test/KillSwitchOracleIntegration.t.sol:133]: Can import Errors and do `bytes(Errors.BORROWING_NOT_ENABLED)`

Comment (lucas-manuel) [reply test/KillSwitchOracleIntegration.t.sol:133]: We should add SDAI and GNO here just to be comprehensive

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: test/KillSwitchOracleIntegration.t.sol (hunk 2 above)
Comment (hexonaut) [reply test/KillSwitchOracleIntegration.t.sol:133]: fixed

**Review by barrutko — DISMISSED** [review]
New changes look good

File: test/KillSwitchOracleIntegration.t.sol (hunk 2 above)
Comment (hexonaut) [reply test/KillSwitchOracleIntegration.t.sol]: missed the top part - also done now

**Review by barrutko — APPROVED** [review]

**Pull request merged**